scripts/env_setup.sh

```diff
@@ -11,6 +11,25 @@ getDependencies(){
     ./get_dependencies.sh
 }
 
+#Get user bot token and store in json file
+getBotToken(){
+	# Prompt user for their bot token
+	read -p "Enter your bot token: " token
```

Comment:
I never knew about the `-p` flag for `read`, that's nice.

Reply:
Let's add a quick error handle check after this `read` command as well. We can do something like...
```
if [ -z "$token" ]; then
   echo "You must enter a Discord Bot token to initialize a Server Duels developer environment."
   exit 1
fi
```